tests/unitTests/unitTest.xml

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
In general, make this as minimal as possible and throw out any lines that are not necessary for the unit test